src/text-input/index.ts

```diff
@@ -52,7 +54,7 @@ export const ThemedBase = ThemedMixin(WidgetBase);
 		'labelAfter',
 		'labelHidden'
 	],
-	attributes: [ 'widgetId', 'label', 'placeholder', 'controls', 'type', 'minLength', 'maxLength', 'value', 'name' ],
+	attributes: [ 'widgetId', 'label', 'placeholder', 'controls', 'type', 'minLength', 'maxLength', 'value', 'name', 'pattern', 'autocomplete' ],
```

Comment:
pattern will have to be a property if it's not just a string.

Reply:
I talked with @matt-gadd today and that doesn't seem to be the case